src/index.html

```diff
@@ -6,9 +6,51 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&amp;display=swap">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+
+      <a href="#" class="logo">
+        <img class="logo" src="images/logo.png" alt="logo"></img>
```

Comment:
```suggestion
        <img class="logo" src="images/logo.png" alt="logo" />
```
Вроде можно self closing делать